examples/generate.cfg

```diff
@@ -3,9 +3,9 @@
 # python generate.py --config examples/generate.cfg
 
 # set output file
-output = examples/scenario.json
+output = examples/scenario_B.json
 # set vehicle types file
-vehicle_types = examples/vehicle_types.json
+vehicle_types = vehicle_types.json
```

Comment:
This is relative to your working directory. As the executable files are in the root folder, this means you'd need the default vehicle_types file in the root folder as well (which I'm against, as it should only serve as an example). Please revert.